docs/pages/install/docker.mdx

```diff
@@ -1,6 +1,6 @@
 # Run Pelican Server with Docker Image
 
-This document explains how to run a Pelican server using Pelican docker image. If you are installing Pelican to use its client functionalities (download or upload an object), refer to [previous sections](../install.mdx) to download and install a Pelican binary instead.
+This document explains how to run a Pelican server using Pelican docker image. If you want to use Pelican client functionalities, such as download or upload an object, refer to [previous sections](../install.mdx) to download and install a Pelican binary instead.
```

Comment:
Currently this comment suggests that you can't use the Docker image for client functionality; I think this is incorrect.

```suggestion
This document explains how to run a Pelican server using Pelican docker image. For Pelican client functionalities, such as download or upload an object, you can use the docker image as well, or you can install the Pelican binary instead as described [elsewhere](../install.mdx).
```

Reply:
They can, but our Docker images aren't set up in a way to facilitate client usage, and we don't recommend this behavior.

Reply:
Ah, got it. 
Totally did not override the entrypoint of the container, add pelican to the PATH, and use that as a client to download a test file. That would be silly..